README.md

```diff
@@ -37,12 +37,15 @@ Math magicians is a website for all fans of mathematics
 ### Tech Stack <a name="tech-stack"></a>
 
 - **ReactJS**
+- **Bootstrap**
 - **CSS 3**
 
 <!-- Features -->
 
```

Comment:
- [ ] As requested by the previous reviewer, All your `back-to-top` links in the README file are not working, Please make sure to fix this issue so they link to the corresponding top navigation section of the readme file.